src/scenarios/billing/ListUserPaymentHistory/ListUserPaymentHistory.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  useScenarioHost,
+  VariableSearch,
+} from '@axinom/mosaic-fe-samples-host';
+import { useState } from 'react';
+import {
+  Container,
+  Divider,
+  Form,
+  Grid,
+  Header,
+  Label,
+  Segment,
+} from 'semantic-ui-react';
+import { getApolloClient } from '../../../apollo-client';
+import { getSubscriptionTransactionsQuery } from './graphql-documents';
+
+export const ListUserPaymentHistory: React.FC = () => {
+  const { activeProfile, logger } = useScenarioHost();
+  const [userAccessToken, setUserAccessToken] = useState<string>();
+
+  const apolloClient = getApolloClient(
+    new URL('graphql', activeProfile.billingServiceBaseURL).href,
+  );
+
+  const fetchTransactionHistory = async (): Promise<void> => {
+    try {
+      const result = await apolloClient.query({
+        query: getSubscriptionTransactionsQuery,
+        context: {
+          headers: {
+            Authorization: `Bearer ${userAccessToken}`,
+          },
+        },
+        fetchPolicy: 'no-cache',
+      });
+
+      logger.log(
+        'method [fetchPaymentPlansAndPaypalSettings]',
```

Comment:
Update this to `fetchTransactionHistory`. Or better still, you could use `method [${fetchTransactionHistory.name}]` to make it dynamic so that we don't have to update these texts if we refactor the name of the function later on